dags/rock/rock_prayer_requests.py

```diff
@@ -0,0 +1,128 @@
+from airflow.models import Variable
+from airflow.hooks.postgres_hook import PostgresHook
+from html_sanitizer import Sanitizer
+import nltk
+from utilities import safeget, get_delta_offset
+from rock.rock_media import is_media_video, is_media_audio
+
+import requests
+
+nltk.download("punkt")
+
+
+class PrayerRequest:
+
+    def __init__(self, kwargs):
+        self.kwargs = kwargs
+        self.headers = {
+            "Authorization-Token": Variable.get(kwargs["client"] + "_rock_token")
+        }
+        self.pg_connection = kwargs["client"] + "_apollos_postgres"
+        self.pg_hook = PostgresHook(
+            postgres_conn_id=self.pg_connection,
+            keepalives=1,
+            keepalives_idle=30,
+            keepalives_interval=10,
+            keepalives_count=5,
+        )
+
+    def get_postgres_person_id(self, personAliasId):
+        personAlias = requests.get(
+            f"{Variable.get(self.kwargs['client'] + '_rock_api')}/PersonAlias/{personAliasId}",
+            params={},
+            headers=self.headers,
+        ).json()
+        rockPersonOriginId = safeget(personAlias, "PersonId")
+        postgresPersonId = self.pg_hook.get_first(
+            "SELECT id FROM people WHERE people.origin_id = %s", (str(rockPersonOriginId),))
+
+        return postgresPersonId[0]
+
+    # "created_at", "updated_at", "text", "person_id", "origin_id", "origin_type", "apollos_type"
+    def map_prayer_requests(self, prayer_request):
+        postgresPersonId = self.get_postgres_person_id(
+            safeget(prayer_request, "RequestedByPersonAliasId"))
+        return (
+            self.kwargs["execution_date"],
+            self.kwargs["execution_date"],
+            safeget(prayer_request, "Text"),
+            postgresPersonId,
+            str(safeget(prayer_request, "Id")),
+            'rock',
+            'PrayerRequest',
+        )
+
+    def run_fetch_and_save_prayer_requests(self):
+        fetched_all = False
+        skip = 0
+        top = 10000
+
+        while not fetched_all:
+            # Fetch prayer requests from Rock
+
+            params = {
+                "$top": top,
+                "$skip": skip,
+            }
+
+            """ if not self.kwargs["do_backfill"]:
+                params["$filter"] = get_delta_offset(self.kwargs) """
+
+            rock_objects = requests.get(
+                f"{Variable.get(self.kwargs['client'] + '_rock_api')}/PrayerRequests",
+                params=params,
+                headers=self.headers,
+            ).json()
+
+            if not isinstance(rock_objects, list):
+                print(rock_objects)
+                print("oh uh, we might have made a bad request")
+                print("top: {top}")
```

Comment:
I've spotted this in a few other places in the app (all my fault) ;) make sure you make these print statements formatted strings
```suggestion
                print(f"top: {top}")
```

Reply:
👍